packages/docs/docs/docs/proxies.md

```diff
@@ -0,0 +1,68 @@
+---
+id: proxies
+title: Proxy Pattern
```

Comment:
I would like to brand this:
The ZeppelinOS Upgrade Pattern

The problem here is that the ideas for this come from many different places, so maybe at the end we can add some generic thanks to everybody researching about upgradeability in ethereum.

@martriay what do you think?

Reply:
Keep in mind that the title has to fit in the sidebar =D